addons/udes_stock/tests/test_stock_location.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from .common import BaseUDES
+
+
+class TestStockLocation(BaseUDES):
```

Comment:
I would add another test with multiple levels: 
```
Stock > Zone 01 > Loc 01-01
Stock > Zone 01 > Loc 01-02
Stock > Zone 02 > Loc 02-01
Stock > Zone 02 > Loc 02-02
```
* Locations Loc 02-02 and Loc 01-01